README.md

```diff
@@ -76,29 +71,27 @@ print(result) // Optional("some value")
 
 ## Documentation
 
-The docs for the latest tagged release are always available at [docs.redistack.info](http://docs.redistack.info).
+The docs for the latest tagged release are always available at the [Swift Package Index][Documentation].
 
 ## Questions
 
-For bugs or feature requests, file a new [issue](https://gitlab.com/mordil/RediStack/-/issues).
-
-For all other support requests, please email [support@redistack.info](mailto:support@redistack.info).
```

Comment:
I agree with removing this support link, but we might benefit from pointing towards the Swift OpenSource slack.

Reply:
Let's do this as a follow up?